src/module/config.ts

```diff
@@ -49,6 +50,7 @@ export type Color =
   | "blue"
   | "orange"
   | "white";
+export type AttackType = "missile" | "melee" | "attack";
```

Comment:
possibly merge from main first and rewrite using the method you created in #369 ?